webcompat/views.py

```diff
@@ -44,8 +43,7 @@
               404: 'Not Found. Lost in Punk Cat Space',
               429: 'Cool your jets! Please wait {0} seconds before making'
                    'another search.',
```

Comment:
One more little thing :)

>>> ERROR_DICT = {400: 'Bad Request.',
...               401: 'Unauthorized. Please log in.',
...               403: 'Forbidden. Maybe that looking at private stuff?',
...               404: 'Not Found. Lost in Punk Cat Space',
...               429: 'Cool your jets! Please wait {0} seconds before making'
...                    'another search.',
...               500: 'Internal Server Error'}
>>> time_left = 60
>>> message = (ERROR_DICT[429]).format(time_left)
>>> message
'Cool your jets! Please wait 60 seconds before makinganother search.'

You need to a trailing space after making or a leading space before another. ^_^